ev3dev2/button.py

```diff
@@ -31,7 +31,7 @@
 import array
 import time
 import evdev
-from ev3dev import get_current_platform
+from ev3dev2 import get_current_platform
```

Comment:
Will this work as `from . import get_current_platform`? Could be future-name-change-proof like this.